crates/worktree/src/worktree.rs

```diff
@@ -125,6 +125,7 @@ pub struct RemoteWorktree {
     background_snapshot: Arc<Mutex<(Snapshot, Vec<proto::UpdateWorktree>)>>,
     project_id: u64,
     client: AnyProtoClient,
+    settings: WorktreeSettings,
```

Comment:
Can we just store the file_scan_inclusions here instead of all the settings?

Reply:
For sure! I think I was trying to get access to the info I needed and just decided to pass down the WorktreeSettings struct to get access to the `is_path_always_included` method, but it probably makes more sense to just pass the strictly needed info down.